sympy/polys/tests/test_groebnertools.py

```diff
@@ -124,6 +124,16 @@ def _do_test_groebner():
         y**8 - 2*y**6 + QQ(3,2)*y**4 - QQ(1,2)*y**2 + QQ(1,16),
     ]
 
+    # https://github.com/sympy/sympy/issues/26682
+    R, s,c = ring("s,c", RR, lex)
```

Comment:
```suggestion
    R, s, c = ring("s,c", RR, lex)
```

Reply:
Have you looked at the rest of the code in this file?